docs/getting-started-guides/network-policy/walkthrough.md

```diff
@@ -6,7 +6,14 @@ assignees:
 
 Kubernetes can be used to declare network policies which govern how Pods can communicate with each other.  This document helps you get started using the Kubernetes [NetworkPolicy API](/docs/user-guide/networkpolicies), and provides a demonstration thereof. 
 
-In this article we assume that a Kubernetes cluster has been created with network policy support.  There are a number of network providers which support NetworkPolicy (see the "Using X for NetworkPolicy" articles in this section).  The reference implementation is [Calico](/docs/getting-started-guides/network-policy/calico) running on GCE.
+In this article, we assume a network provider that supports the NetworkPolicy API has been installed. Network providers can be installed as add-ons using [kubeadm](/docs/getting-started-guides/kubeadm/), or any other suitable cluster installer.
```

Comment:
I think this paragraph conflates `kubeadm`, which is a tool for deploying Kubernetes, and `add-ons` which are ways of deploying things on top of Kubernetes.  Deploying a network via an add-on doesn't actually depend on `kubeadm`.